src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -383,8 +363,8 @@ export const WithDescription = {
         return (
             <Dropdown
                 {...args}
-                label={currentOption.label}
-                value={currentOption.value}
+                label={currentOption?.label || 'Select something'}
```

Comment:
- You can remove label
- You can use `placeholder` to set `Select Something`
- Use `value={currentOption?.value}`